common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -65,23 +75,23 @@
 from django.template.loaders.filesystem import Loader
 engine = Engine(dirs=settings.DEFAULT_TEMPLATE_ENGINE['DIRS'])
 source, template_path = Loader(engine).load_template_source(path)
-%>${source}</%def>
+%>${source | n, decode.utf8}</%def>
 
 <%def name="require_module(module_name, class_name)">
     <script type="text/javascript">
         (function (require) {
             % if settings.REQUIRE_DEBUG:
-                require(['${module_name}'], function (${class_name}) {
-                    ${caller.body()}
+                require(['${module_name | n, js_escaped_string}'], function (${class_name | n, js_escaped_string}) {
```

Comment:
Seems like `| n, decode.utf8` is the best option here.  Is there anything else needed?